### ISSUE TITLE
Consider shields for US county routes on blade signs

### ISSUE THREAD

**Issue by claysmalley** [ZeLonewolf/openstreetmap-americana#686]
Some counties in the United States number their routes, but have no shields, choosing instead to post route numbers on blade signs. In some cases, these route numbers exist alongside a local street name, like East Kercher Road and Elkhart County Road 38 in Goshen, Indiana:

[<img width=400 alt="Screenshot from 2023-01-13 13-35-14" src="https://user-images.githubusercontent.com/1732117/212394054-584ecbda-5a9a-47ef-89c9-62e1ebf713d4.png" />](https://www.bing.com/maps?toWww=1&redig=63687871B4A94CB3AFF82C17B2EC8B25&cp=41.551508%7E-85.78376&lvl=19.9&mo=om.1&style=x)

In other localities, like Liberty County, Texas, blade signs may have a county road number and nothing else, indicating that the route number is the sole identity of the road:

[<img width=400 alt="Screenshot from 2023-01-13 13-43-42" src="https://user-images.githubusercontent.com/1732117/212395467-f0c571b4-f6e2-4ab3-981f-d1ef43ec4a01.png" />](https://www.bing.com/maps?toWww=1&redig=63687871B4A94CB3AFF82C17B2EC8B25&cp=30.305168%7E-95.110759&lvl=19.8&mo=om.1&style=x)

Currently, routes belonging to these counties show up as a bare number with no shield background. One possible treatment would be to assign white-on-green (or other local color scheme) rectangular shields, and preprocess `ref` to prepend "CR" (or other local initialism) and a space.

This would fill in visual gaps in states with a lot of internal variation in county route signage, though it may not be appropriate nationwide, especially in states that are completely free of shielded county routes.

**Comment by ZeLonewolf** [comment]
Thanks for the summary.  I'd like to see the solution (whatever we decide) fit within the following principles:
* Whatever is decided for one system of blade signs, should be applied to all systems that are of the same type.
* The routes should really be equivalent to county route systems in other places, as opposed to merely numbered streets such as "5th Ave"

If we choose to render a graphic as describe, consider perhaps a reduced font size and tight margins, possibly with no border color, so as to be analogous to the blade.

**Comment by zekefarwell** [comment]
I don't think we should render a shield style icon for every road that happens to be numbered.  The point of showing a shield style icon on the map is to show that there are shield style signs in the real world.  If there are just normal blade street signs we should label the road the same way as any named road, with text aligned along the way.  

The problem here seems to be that these roads are mapped with route relations that have a `network` and a `ref` and Americana interprets this as meaning there are shields, but that's not true.  So either we need to reconsider that assumption and figure out a way to exclude route relations that don't use shields, or these roads shouldn't be mapped as route relations.  Not sure which is the right answer in this case.

**Comment by StretchLongfellow** [comment]
I agree with zekefarwell on both accounts. Nebraska is a great example of how this has gone wrong; look at other maps for examples. All numbered county roads wound up with shields because they're named CR XX, but named roads did not. (OSM used to do the same until I cleaned it up...)

Would it be possible to add a shield=yes tag to relations?

**Comment by 1ec5** [comment]
In Ohio, each county and township decides how to signpost its own roads along its own roads. About half of the counties along with their townships post shields, which this style displays. The rest rely on blade signs that may or may not indicate a road number. Whether or not a particular county posts shields, the state MUTCD calls for cross street signs along state highways to append any road number onto the road name, with a little caption above to clarify the network.

Applying a “shield” to the county and township route relations would polka-dot the map in green badges. Literally every locally-maintained road outside of town would have one, even some gravel driveways. All it would communicate to the reader is that street name signs in this part of the world have a green background, duh.

>  If there are just normal blade street signs we should label the road the same way as any named road, with text aligned along the way.

This would match the style of signage that ODOT posts when it doesn’t want to show a shield. For example, Brown and Adams counties do not post shields but do post county road and township road numbers along local roads. So the state highways have signs like this:

[![C-24](https://user-images.githubusercontent.com/1231218/223844353-ac61f713-8445-46c6-b889-e88a9df6d7a6.png)](https://www.mapillary.com/app/?pKey=1745169489024809)

[![T-48](https://user-images.githubusercontent.com/1231218/223846592-738aa462-0781-4548-a47b-d3a87cd9dcbb.png)](https://www.mapillary.com/app/?pKey=107965818043891)

Scioto County does post shields (the distinctive orange ones), but the same plain-text signs are present too:

[![C-30](https://user-images.githubusercontent.com/1231218/223853647-82f57b5e-ba81-4637-bcd0-82ce86f0cf3d.png)](https://www.mapillary.com/app/?pKey=2248572995276058)

In order to achieve an effect in the road label, we’d need to prepend “C-” or “T-” to the relation’s `ref`, similar to what was rejected in #141. We could introduce a county-by-county lookup table, but that would get unwieldy. [Wikidata has a ref formatter property](https://www.wikidata.org/wiki/Property:P8498) on each of the network items specifically for this purpose. Alternatively, we could relax our enthusiasm for route relations and get the already formatted route number from the `ref` tag on each way. (But we’d have to somehow determine that it wouldn’t be redundant to the route relations the way is a member of.)
